doc-content/enterprise-only/installation/kie-server-thread-efficiency.adoc

```diff
@@ -0,0 +1,18 @@
+[id='kie-server-thread-efficiency-con_{context}']
```

Comment:
Sorry to nit. The file name should match the ID, or in anycase include the module type, so kie-server-thread-efficiency-con.adoc

Reply:
Keep nitting. These things need to be spotted.